<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Start</class>
 <widget class="QFrame" name="Start">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>252</width>
    <height>228</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="create_map">
     <property name="text">
      <string>创建地图</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="load_map">
     <property name="text">
      <string>载入地图</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="create_room">
     <property name="text">
      <string>创建房间</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="enter_room">
     <property name="text">
      <string>加入房间</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="log_out">
     <property name="text">
      <string>退出登录</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
